recipe/meta.yaml

```diff
@@ -24,65 +24,68 @@ outputs:
         - napari = napari.__main__:main
 
     requirements:
-      build:
-        - pep517
-        - setuptools_scm
       host:
-        - python >=3.8
+        - python >=3.9
         - pip
-        - setuptools_scm
+        - setuptools >=69
+        - setuptools_scm >=8
       run:
-        - python >=3.8
+        - python >=3.9
 
-        # dependencies matched to setup.cfg
+        # dependencies matched to pyproject.toml
+        - app-model >=0.2.5,<0.3.0a
         - appdirs >=1.4.4
-        - app-model >=0.1.2,<0.3.0a
         - cachey >=0.2.1
         - certifi >=2018.1.18
         # Do not depend on base "dask" package since it pulls
         # in the distributed dependency
         # conda-forge doesn't have the dask-array subpackage
         # so depend on cytoolz and numpy manually
-        - dask-core>=2.15.0,!=2.28.0
+        - dask-core >=2021.10.0
         - cytoolz >=0.11.0
         - imageio >=2.20,!=2.22.1
         - jsonschema >=3.2.0
         - lazy_loader >=0.2
-        - magicgui >=0.3.6
+        - magicgui >=0.7.0
         - napari-console >=0.0.9
         - napari-plugin-engine >=0.1.9
         - napari-svg >=0.1.8
-        - npe2 >=0.7.2
-        - numpy >=1.21,<2.0a0
+        - npe2 >=0.7.6
+        - numpy >=1.22.2
         - numpydoc >=0.9.2
         - pandas >=1.3.0
-        - pillow !=7.1.0,!=7.1.1
+        - pillow >=9.0
         - pint >=0.17
         - psutil >=5.0
-        - psygnal>=0.3.4
+        - psygnal >=0.5.0
         - pydantic >=1.9.0
         - pygments >=2.6.0
         - pyopengl >=3.1.0
         - pyyaml >=5.1
         - qtpy >=1.10.0
         - scikit-image >=0.19.1
         - scipy >=1.5.4
-        - superqt >=0.4.1
-        - tifffile >=2020.2.16
+        - superqt >=0.6.7
+        - tifffile >=2022.4.8
         - toolz >=0.10.0
         - tqdm >=4.56.0
         - typing_extensions >=4.2.0
-        - vispy >=0.14.1,<0.15
+        - vispy >=0.14.1,<0.15.0a0
         - wrapt >=1.11.1
 
         # additional dependencies for convenience in conda-forge
         - fsspec
         - pooch >=1.3.0
-        - zarr
+        - zarr >=2.12.0
 
       run_constrained:
-        - pyside2 >=5.14.2,!=5.15.0
-        - pyqt >=5.12.3,!=5.15.0
+        - napari-plugin-manager >=0.1.0a2,<0.3.0a0
+        - pyside2 >=5.13.2,!=5.15.0
+        - pyside6 <6.5,>=6.7
```

Comment:
i don't think this constraint makes sense. it means that pyside6 is not installable with napari since it must be less than 6.5 and greater than 6.7

Reply:
```
$ mamba create --name np --channel conda-forge --override-channels napari=0.5.0 pyside6

Looking for: ['napari=0.5.0', 'pyside6']

conda-forge/linux-64                                        Using cache
conda-forge/noarch                                          Using cache
Could not solve for environment specs
The following packages are incompatible
├─ napari 0.5.0**  is installable and it requires
│  └─ pyside6 <6.5,>=6.7 , which can be installed;
└─ pyside6 is not installable because it conflicts with any installable versions previously reported.
```